tests/test_segmentation_tools.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import pytest
+from scipy import ndimage
+
+from invesalius.data.mask import Mask
+from invesalius.data.slice_ import Slice
+from invesalius_cy import floodfill
+
+
+def test_region_growing_threshold():
+    # Dummy Image
+    image = np.array(
+        [
+            [1, 1, 1, 5, 5],
+            [1, 2, 2, 5, 5],
+            [1, 2, 3, 5, 5],
+            [1, 2, 2, 5, 5],
+            [1, 1, 1, 5, 5],
+        ],
+        dtype=np.int16,
+    )
```

Comment:
You can create a 3d image with a single slice this way:

```python
    image = np.array(
       [[
            [1, 1, 1, 5, 5],
            [1, 2, 2, 5, 5],
            [1, 2, 3, 5, 5],
            [1, 2, 2, 5, 5],
            [1, 1, 1, 5, 5],
        ]],
        dtype=np.int16,
    )
```

So `image_3d = image[np.newaxis, ...]` is not necessary anymore.